portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+public class Comment {
+    private long id;
+    private String name;
+    private String email;
+    private String subject;
+    private String message;
+    private long timestamp;
+
+    public Comment(long id, String name, String email, String subject, String message, long timestamp) {
+        this.id = id;
+        this.name = name;
+        this.email = email;
+        this.subject = subject;
+        this.message = message;
+        this.timestamp = timestamp;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public String getSubject() {
+        return subject;
+    }
+
+    public String getMessage() {
+        return message;
+    }
```

Comment:
nit: these getters aren't actually used right? If not, I'd just remove them. See go/yagni